packages/module/patternfly-docs/content/extensions/chatbot/examples/demos/Chatbot.md

```diff
@@ -103,3 +103,11 @@ This demo displays an embedded ChatBot. Embedded ChatBots are meant to be placed
 ```js file="./EmbeddedChatbot.tsx" isFullscreen
 
 ```
+
+### Comparing ChatBots
+
+To let users compare how different ChatBots respond to the same prompt, you can add multiple ChatBots within the same window. The following demo illustrates a comparison view pattern that allows users to toggle between different conversations in a single ChatBot window.
```

Comment:
```suggestion
To let users compare how different ChatBots respond to the same prompt, you can add multiple ChatBots within the same window. This demo illustrates a comparison view pattern that allows users to toggle between different conversations in a single ChatBot window.
```
small thing I noticed - the other demos use "this demo" rather than "the following demo", so I'm being picky to match lol

Reply:
All set! 💃